src/screens/Tag.js

```diff
@@ -136,6 +136,24 @@ const TagPage = () => (
           <GenericGridArea />
         </Property>
 
+        <Property name="messages">
+          <Description>
+            Custom messages for remove button on Tag. Used for
+            internationalization and customization of default strings and
+            accessibility announcements.
+          </Description>
+          <PropertyValue type="string">
```

Comment:
Should this be object instead of string?
```suggestion
          <PropertyValue type="object">
```